packages/core/src/base-chart.ts

```diff
@@ -394,6 +394,8 @@ export class BaseChart {
 			chartId = this.id;
 			container = parent.append("div");
 			container.attr("chart-id", chartId)
+				// Alert user when the contents of the chart container changes
+				.attr("role", "alert")
```

Comment:
Not sure if this is the greatest choice. If multiple charts are on the page, then the user could potentially be receiving too many alerts. @zvonimirfras @cal-smith what do you think?

Reply:
[based on MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role) this definitely wouldn't be a good use of `role="alert"`

Reply:
Great thanks Cal